packages/components/src/components/Tabs/Tabs.js

```diff
@@ -19,7 +19,7 @@ import {
 } from 'carbon-components-react';
 
 const TabContent = ({ selected, ...other }) =>
-  selected && <CarbonTabContent selected={selected} {...other} />;
+  selected ? <CarbonTabContent selected={selected} {...other} /> : <span style="visibility: hidden;"></span>;
```

Comment:
Still missing the id on the new span or is it not required?

Reply:
Thanks, the `id` attribute was important. I've added the `id` and the violation is gone. (Adding visibility style didn't have any impact but still left it there just to be on the safer side) This is what I get after manipulating the elements from the Chrome dev tool. Let's see if the changes in the package component work as well. 

![Screen Shot 2022-11-22 at 2 07 42 PM](https://user-images.githubusercontent.com/44249850/203401310-f6bed54b-ec71-4285-a3e2-2621286637e8.png)